glade_forms/lib/src/validator/validator_result/glade_validator_result.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:equatable/equatable.dart';
+import 'package:glade_forms/src/core/error/error_serverity.dart';
+import 'package:glade_forms/src/core/error/glade_input_error.dart';
+
+/// When validation failed but we already have propert [T] value.
+typedef OnValidate<T> = String Function(T value);
+
+abstract class GladeValidatorResult<T> extends GladeInputError<T> with EquatableMixin {
+  /// Error message when translation is not used. Useful for development.
+  final OnValidate<T> onValidateMessage;
+
+  /// Value which triggered validation and returned this error.
+  final T value;
+
+  final ErrorServerity _errorServerity;
+
+  @override
+  ErrorServerity get severity => _errorServerity;
+
+  @override
+  String get error => onValidateMessage(value);
+
+  String get devErrorMessage => onValidateMessage(value);
```

Comment:
Maybe change "error" since it represents both error and warning? Into i.e. "validation"?

Reply:
On validator() level we still have devError. Renaming will be breaking change